visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -75,10 +76,10 @@ export class CodeChartaController {
 			.then(() => {
 				this.storeService.dispatch(setState(ScenarioHelper.getDefaultScenarioSetting()))
 			})
-			.catch(e => {
+			.catch((errors: ErrorObject[]) => {
 				this.storeService.dispatch(setIsLoadingFile(false))
-				console.error(e)
-				this.printErrors(e)
+				console.error(errors)
+				this.dialogService.showErrorDialog(JSON.stringify(errors))
```

Comment:
Stringifying errors is going to loose some information depending on how the object looks like. I do not know where this is logged though and what tools you otherwise use to inspect objects.

Reply:
mdDialog expects it's content as a string

Reply:
That is fine :-) what I meant is that it's best to use something else than `JSON.stringify()` to get the string representation of the value.